packages/wallet/src/cip30.ts

```diff
@@ -74,6 +74,15 @@ const compareUtxos = (utxo: Cardano.Utxo, comparedTo: Cardano.Utxo) => {
   return 0;
 };
 
+const cardanoAddressToCbor = (address: Cardano.Address | Cardano.RewardAccount): Cbor =>
+  usingAutoFree((scope) => {
+    const cmlAddr = parseCmlAddress(scope, address.toString());
```

Comment:
I am not 100% sure this is the right way to do it for reward accounts too. @mkazlauskas, do you know if this is correct?

Reply:
Yes this seems to be the correct way to do it for reward accounts too. Would be best to test it e2e with a dapp

Reply:
added e2e test in [cda9ceb](https://github.com/input-output-hk/cardano-js-sdk/pull/599/commits/cda9ceb71622602d026fd3ba637637d419d14d31)